src/System.IO.FileSystem.AccessControl/tests/DirectoryObjectSecurityTests.cs

```diff
@@ -0,0 +1,232 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+using System.DirectoryServices;
+using System.Security.Principal;
+using Xunit;
+
+namespace System.Security.AccessControl
+{
+    public class DirectoryObjectSecurityTests
+    {
+        [Fact]
+        public void ObjectInitialization_DefaultConstructor_Success()
+        {
+            var activeDirectorySecurity = new ActiveDirectorySecurity();
+            Assert.NotNull(activeDirectorySecurity);
```

Comment:
We don't need to assert that the result of a new operation is non-null. Instead we should assert that the various properties of the object match expected defaults (and if such tests exist elsewhere, this one could just be deleted).